taipy/gui/utils/callable.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021-2024 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+import typing as t
+from inspect import isclass
+from types import LambdaType
+
+
+def _is_function_like(s: t.Any) -> bool:
```

Comment:
I would have called this  '_is_function' to be honest.
... and... can a class be callabled? Seriously??

Reply:
Why not
of course, that's why I switched to isroutine